File: app/src/main/java/net/squanchy/schedule/view/ScheduleSpeakerView.java
package net.squanchy.schedule.view;

import android.content.Context;
import android.util.AttributeSet;
import android.view.ViewGroup;
import android.widget.ImageView;

import net.squanchy.R;
import net.squanchy.support.widget.SpeakerView;

public class ScheduleSpeakerView extends SpeakerView {

    public ScheduleSpeakerView(Context context, AttributeSet attrs) {
        this(context, attrs, 0);
    }

    public ScheduleSpeakerView(Context context, AttributeSet attrs, int defStyleAttr) {
        this(context, attrs, defStyleAttr, 0);
    }

    public ScheduleSpeakerView(Context context, AttributeSet attrs, int defStyleAttr, int defStyleRes) {
        super(context, attrs, defStyleAttr, defStyleRes);
    }

    @Override
    protected ImageView inflatePhotoView(ViewGroup speakerPhotoContainer) {
        return (ImageView) layoutInflater().inflate(R.layout.view_speaker_photo_schedule, speakerPhotoContainer, false);
    }
}

File: app/src/main/java/net/squanchy/schedule/view/ScheduleSpeakerView.kt
package net.squanchy.schedule.view

import android.content.Context
import android.util.AttributeSet
import android.view.ViewGroup
import android.widget.ImageView

import net.squanchy.R
import net.squanchy.support.widget.SpeakerView

class ScheduleSpeakerView @JvmOverloads constructor(
    context: Context,
    attrs: AttributeSet,
    defStyleAttr: Int = 0,
    defStyleRes: Int = 0
) : SpeakerView(context, attrs, defStyleAttr, defStyleRes) {

    override fun inflatePhotoView(speakerPhotoContainer: ViewGroup): ImageView =
        layoutInflater.inflate(R.layout.view_speaker_photo_schedule, speakerPhotoContainer, false) as ImageView
}
